pandas/core/reshape/tile.py

```diff
@@ -372,7 +379,11 @@ def _preprocess_for_cut(x):
         series_index = x.index
         name = x.name
 
-    x = np.asarray(x)
```

Comment:
you can convert it then simply check the ndim <= 1 no?

Reply:
`np.asarray` here will strip tz information if a Series or DatetimeIndex is passed, so we should convert if it's not a numpy or pandas object.

```
In [3]: tz = 'UTC'

In [4]: s = pd.Series(pd.date_range('20130101', periods=3, tz=tz))

In [5]: s
Out[5]:
0   2013-01-01 00:00:00+00:00
1   2013-01-02 00:00:00+00:00
2   2013-01-03 00:00:00+00:00
dtype: datetime64[ns, UTC]

In [6]: np.asarray(s)
Out[6]:
array(['2013-01-01T00:00:00.000000000', '2013-01-02T00:00:00.000000000',
       '2013-01-03T00:00:00.000000000'], dtype='datetime64[ns]')
```

Reply:
so in that case, prefer to do not do this here, rather move any logic needed to ``_convert_bin_to_numeric_type``

Reply:
Well the main issue is that `_preprocess_for_cut` is called first in both `cut` and `qcut`, and it drops the timezone dtype when trying to convert the input array to a numpy structure before the dtype can be set to a variable in the next function `_coerce_to_type`

`_convert_bin_to_numeric_type` looks to be only applicable for `cut` and since it's for converting bins.